package.json

```diff
@@ -11,10 +11,11 @@
     "node": ">=14.0.0"
   },
   "dependencies": {
-    "typescript": "^5.0.4",
-    "@types/express": "^4.17.17",
     "@types/content-disposition": "^0.5.5",
-    "express": "^4.18.2"
+    "@types/express": "^4.17.17",
+    "dotenv": "^16.0.3",
```

Comment:
`dotenv-safe` would be more appropriate. It ensures that env variables are really defined.